src/core_plugins/kibana/server/lib/export/collect_dashboards.js

```diff
@@ -0,0 +1,21 @@
+import collectPanels from './collect_panels';
+
+export const deps = {
```

Comment:
Just so you know, when you use named exports you can stub dependencies like so:

dep.js
```js
export function collectPanels() {
  // ...
}
```

use.js
```js
import { collectPanels } from './dep';
```

test.js
```js
import sinon from 'sinon'

import * as depNS from '../dep';
sinon.stub(depNS, 'collectPanels').returns(...)
```